src/extension/wps/wps-core/src/test/java/org/geoserver/wps/GetCapabilitiesTest.java

```diff
@@ -40,6 +46,22 @@ public void testGetDisabledService() throws Exception {
             gs.save(info);
         }
     }
+    
+    @Test
+    public void testWPSScriptEnable() throws Exception {
```

Comment:
you may wish to rename this test; as you have not really managed to configure an invalid process group